packages/opentelemetry-node/test/registration.test.ts

```diff
@@ -104,8 +101,7 @@ describe('API registration', () => {
 
     assert.ok(context['_getContextManager']() instanceof DefaultContextManager);
 
-    const apiTracerProvider = trace.getTracerProvider();
-    assert.ok(apiTracerProvider instanceof ProxyTracerProvider);
+    const apiTracerProvider = trace.getTracerProvider() as ProxyTracerProvider;
```

Comment:
Why is this needed? ProxyTracerProvider should be still exported as class by API.

Reply:
I wasn't able to figure out why it was needed yesterday. I thought it might have had multiple instances of the API module installed but in my local i was seeing the same behavior and my node_modules didn't have a nested API. My best guess is that either the typescript output has somehow mangled the instanceof check or the test infrastructure is messing with it.

Reply:
You can see the test failure here https://github.com/open-telemetry/opentelemetry-js/runs/1930234943

Reply:
Actually it seems like there are multiple versions of the API. Previously when it was in lerna they were all symlinked to the `packages/opentelemetry-api` but now each module has its own copy of the api module in its node_modules folder

Reply:
Yes, seems to be a side effect of linking.
e.g. api is inside opentelemetry-tracing and opentelemetry-node.
`NodeTracerProvider#register` goes further to `BasicTracerProvider#register` in tracing package. So the `ProxyTracerProvider` created is from API in tracing but test asks if it is an `instanceof` that one in package node.
`instanceof` is not using duck typing so it doesn't match.
before both were linked to the one and only API package in this repo.